sockeye/output_handler.py

```diff
@@ -140,3 +115,29 @@ def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.i
                              t_input.tokens,
                              t_output.tokens,
                              self.threshold)
+
+
+def get_output_handler(output_type: str,
```

Comment:
Why do you need to move this down? This forward declaration of return type works for many other places in the code (see attention.py, decoder.py, encoder.py). I also like to have the get function first to see what is available.